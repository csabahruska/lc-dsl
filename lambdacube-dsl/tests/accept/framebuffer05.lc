render _ =
    let frame = FrameBuffer (DepthImage #1 1.0)
    in ScreenOut frame
main = render (Uni (IM44F "MVP"))
