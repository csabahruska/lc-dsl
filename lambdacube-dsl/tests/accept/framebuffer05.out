Pipeline {backend = OpenGL33, textures = [], samplers = [], targets = [RenderTarget {renderTargets = [(Depth,Just (Framebuffer Depth))]}], programs = [], slots = [], commands = [SetRenderTarget 0,ClearRenderTarget [(Depth,VFloat 1.0)]]}